### ISSUE TITLE
Add check for missing permissions

### ISSUE THREAD

**Issue by JanikCodes** [JanikCodes/minecraft_discord_py#13]
Some inputs can result in an error when the bot does not have enough permissions, add a check for valid permissions or find another way

**Comment by Scarlet-2** [comment]
Hello, i am new on github and contributing and i would like to help

**Comment by JanikCodes** [comment]
Yeah! Basically we need a check in every Slash Command if the bot has the required permissions.
If not, perhaps show an warning embed.

**Comment by Scarlet-2** [comment]
like a automatic message on a channel?

**Comment by JanikCodes** [comment]
Yeah, when ever the command is executed, instead of doing what the command should do, display an error embed.
To contribute I suggest forking this repository and once done creating a Pull Request.

**Comment by JanikCodes** [comment]
Surething! Appreciate it.

**Comment by Scarlet-2** [comment]
Do we put the bot as admin? or the is some specific permission?

**Comment by Scarlet-2** [comment]
And i am having some problems setting up the testing environment

**Comment by JanikCodes** [comment]
> Do we put the bot as admin? or the is some specific permission?

The bot has specific permissions, you can see what permissions we needs in his invite link:
https://discord.com/oauth2/authorize?client_id=1098581527446700083&permissions=2147871745&scope=bot

**Comment by JanikCodes** [comment]
> And i am having some problems setting up the testing environment

what's your problem?
Please have a look at our README.md

**Comment by xGurnoor** [comment]
Any more information on this? Which specific inputs result in an error?

**Comment by JanikCodes** [comment]
> Any more information on this? Which specific inputs result in an error?

Sadly no. I just saw in the logs a while ago that an error occurred in any command, probably because someone invited the bot without ticking all permissions.

I think we have 2 options, add a permission check for every command or I think I can configure in Discord Developer Portal that the bot is required to have all permissions, not sure if that exists.
